### ISSUE TITLE
Could it work on subsequent errors?

### ISSUE THREAD

**Issue by localhostdotdev** [yuki24/did_you_mean#120]
e.g.

- `irb`
- `exi` -> `did you mean ...`
- `exi` -> regular error

Full log:

```
>> exi
Traceback (most recent call last):
        4: from /.rvm/rubies/ruby-head/bin/irb:23:in `<main>'
        3: from /.rvm/rubies/ruby-head/bin/irb:23:in `load'
        2: from /.rvm/rubies/ruby-head/lib/ruby/gems/2.7.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        1: from (irb):2
NameError (undefined local variable or method `exi' for main:Object)
Did you mean?  exit
>> exi
Traceback (most recent call last):
        5: from /.rvm/rubies/ruby-head/bin/irb:23:in `<main>'
        4: from /.rvm/rubies/ruby-head/bin/irb:23:in `load'
        3: from /.rvm/rubies/ruby-head/lib/ruby/gems/2.7.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        2: from (irb):3
        1: from (irb):3:in `rescue in irb_binding'
NameError (undefined local variable or method `exi' for main:Object)
```

I think I'm using `1.3.0`.

**Comment by yuki24** [comment]
Oh interesting, seems like bug. Thanks for reporting!

**Comment by yuki24** [comment]
released as part of [`v1.3.1`](https://github.com/yuki24/did_you_mean/releases/tag/v1.3.1).
